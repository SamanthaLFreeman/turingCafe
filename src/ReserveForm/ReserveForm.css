form {
  display: flex;
  justify-content: space-around;
  width: 90vw;
}

.reserve-btn {
  background-color: teal;
  border: solid 1px #fff;
  color: #fff;
  font-size: 1em;
  padding: 4px;
  width: 12vw;
}

input {
  font-size: 1em;
  padding: 3px;
  width: 12vw;
}
