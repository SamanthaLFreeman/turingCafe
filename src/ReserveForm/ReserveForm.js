import React from 'react';
import './ReserveForm.css';

const ReserveForm = () => {
  return (
    <form>
      <input 
        placeholder='Name'
        name='name'
        value='' />
      <input
        placeholder='Date (mm/dd)'
        name='date'
        value='' />
      <input
        placeholder='Time'
        name='time'
        value='' />
      <input
        placeholder='Number of guests'
        name='guests'
        value='' />
      <button className='reserve-btn'>Make Reservation</button>
    </form>
  )
}

export default ReserveForm;