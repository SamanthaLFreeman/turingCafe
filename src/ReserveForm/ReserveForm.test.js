import React from 'react';
import { shallow } from 'enzyme';
import ReserveForm from './ReserveForm';

describe('ReserveForm', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <ReserveForm />)
  })

  it('should match the snapshot with all of the data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  })
})