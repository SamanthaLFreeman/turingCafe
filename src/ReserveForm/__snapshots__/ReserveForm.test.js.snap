// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReserveForm should match the snapshot with all of the data passed through 1`] = `
<form>
  <input
    name="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
    value=""
  />
  <input
    name="date"
    onChange={[Function]}
    placeholder="Date (mm/dd)"
    type="text"
    value=""
  />
  <input
    name="time"
    onChange={[Function]}
    placeholder="Time"
    type="text"
    value=""
  />
  <input
    name="number"
    onChange={[Function]}
    placeholder="Number of guests"
    type="number"
    value=""
  />
  <button
    className="reserve-btn"
    onClick={[Function]}
  >
    Make Reservation
  </button>
</form>
`;
